src/System.Text.Encodings.Web/src/System.Text.Encodings.Web.csproj

```diff
@@ -31,4 +32,11 @@
     <Reference Include="System.Memory" />
     <Reference Include="System.Resources.ResourceManager" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetsNetCoreApp)' == 'true'">
```

Comment:
Adding this back since we now need the netcoreapp config. It was removed in  the following PR when the UAP target got removed: https://github.com/dotnet/corefx/pull/41759

cc @ViktorHofer

Reply:
If you want to multi-target again, sure.